lib/cli.js

```diff
@@ -255,7 +255,7 @@ const cli = {
         }
 
         if (options.printConfig) {
-            if (files.length) {
+            if (files.length > 1) {
```

Comment:
`--print-config` is a string option:

https://github.com/eslint/eslint/blob/278813a6e759f6b5512ac64c7530c9c51732e692/lib/options.js#L311-L315

When you type `eslint --print-config foo.js`, `foo.js` isn't interpreted as a positional argument (`files`), but as the value of the `--print-config` option.

`options.printConfig` itself is the `"foo.js"` string.

Reply:
I see now, nvm then, closing. I had a `file` before the parameter.